tox.ini

```diff
@@ -15,15 +15,10 @@ passenv =
   TRAVIS_JOB_NUMBER
   TRAVIS_PULL_REQUEST
   TRAVIS_REPO_SLUG
-deps =
-    pytest
-    freezegun
-    !py36: vdirsyncer
-    py36: vdirsyncer==0.16.8
-    python-dateutil
```

Comment:
This one is also missing, but is not required because it is a non-test dependency already.

Reply:
(nothing to do on this item, just FYI)